api/src/main/java/org/pmiops/workbench/interceptors/TracingInterceptor.java

```diff
@@ -25,17 +37,36 @@
   private static final Tracer tracer = Tracing.getTracer();
   private static final Logger log = Logger.getLogger(FirecloudApiClientTracer.class.getName());
   private static final String TRACE_ATTRIBUTE_KEY = "Tracing Span";
+  private static final StatsRecorder STATS_RECORDER = Stats.getStatsRecorder();
```

Comment:
nit: if we go this route, let's wrap this in a `Provider<StatsRecorder>`. That will allow us to mock it and capture arguments.